include/wavm/WAVMWasmModule.h

```diff
@@ -194,6 +192,8 @@ class WAVMWasmModule final
     void prepareOpenMPContext(const faabric::Message& msg);
 
     std::unique_ptr<openmp::PlatformThreadPool> OMPPool;
+
+    uint32_t createMemoryGuardRegion();
```

Comment:
For a while I thought ther was some sort of memory corruption issue, so adding guard regions around the dynamic modules makes sense. This way we should be able to catch stack-overflows in those modules.